src/components/message-input/index.tsx

```diff
@@ -398,12 +398,14 @@ export class MessageInput extends React.Component<Properties, State> {
 
               <div className='message-input__emoji-icon-outer'>
                 <div className='message-input__icon-wrapper'>
-                  <IconButton
-                    className={classNames('message-input__icon', ' message-input__icon--emoji')}
-                    onClick={this.openEmojis}
-                    Icon={IconFaceSmile}
-                    size={24}
-                  />
+                  {!this.props.isEditing && (
```

Comment:
Place here so the layout of the message input is not affected by completely removing this element.